examples/controllers/src/backend/app/core.clj

```diff
@@ -0,0 +1,32 @@
+(ns core
+  (:require
+    [framework.coercion.core :as coercion]
+    [framework.config.core :as config]
+    [framework.interceptor.core :as xiana-interceptors]
+    [framework.route.core :as routes]
+    [framework.webserver.core :as ws]
+    [interceptors]
+    [piotr-yuxuan.closeable-map :refer [closeable-map]]
+    [route]))
+
+(defn ->system
+  [app-cfg]
+  (-> (config/config)
+      (merge app-cfg)
+      routes/reset
+      ws/start
+      (select-keys [:routes
+                    :controller-interceptors
+                    :webserver])
+      closeable-map))
```

Comment:
core/system doesn't need to be closeable

Reply:
see above